modules/custom/d_demo/pages/about_us.yml

```diff
@@ -84,7 +84,7 @@ field_page_section:
                 target: _self
         field_d_settings:
           type: field_p_configuration_storage
-          custom_class: theme-invert
+          custom_class: theme-invert theme-default padding-top-default padding-bottom-default margin-top-default margin-bottom-default
```

Comment:
nie jestem pewien czy to sie dobrze skonczy jak beda oba `.theme-invert` oraz `.theme-default`
docelowo .theme-invert ma byc wykasowane.. ale chyba bym tu teraz nie dodawał .theme-default .. chyba, ze bez tego cos sie psuje ?